src/translate/java/org/javarosa/engine/xml/XmlUtil.java

```diff
@@ -41,7 +41,13 @@ public static String getPrettyXml(byte[] xml) {
             DOMConfiguration domConfig = lsSerializer.getDomConfig();
             domConfig.setParameter("format-pretty-print", true);
             domConfig.setParameter("cdata-sections", true);
-            lsSerializer.setNewLine("\n");
+
+            // LSSerializer does not have an explicit control for ending the XML Declaration with a
+            // newline, hence the need for the property isStandalone
+            // More in https://bugs.openjdk.org/browse/JDK-8259502
+            domConfig.setParameter("http://www.oracle.com/xml/jaxp/properties/isStandalone", true);
```

Comment:
Does this property add or remove new lines ? From the description it feels like it adds a new line, but I see that we have removed new lines from `pretty_printed_xml`.

Reply:
It just adds a new line after the xml declaration. The removed lines are from deleting newlines between namespaces, it seems that that format is not applied by the `format-pretty-print` property.